pkg/master/master_test.go

```diff
@@ -526,7 +526,7 @@ func testInstallThirdPartyAPIListVersion(t *testing.T, version string) {
 			setupEtcdList(fakeClient, "/ThirdPartyResourceData/company.com/foos/default", test.items)
 		}
 
-		resp, err := http.Get(server.URL + "/apis/company.com/" + version + "/namespaces/default/foos")
+		resp, err := http.Get(server.URL + "/apis/company.com/group/" + version + "/namespaces/default/foos")
```

Comment:
I remember in a meeting with @bgrant0607 and @lavalamp, we agreed the group would be part of the company.com, i.e., /apis/group.company.com, which is consistent with the URL of other groups, i.e., apis/group/version/... Is there some new development? Thanks.